### PR TITLE
add support for newer zaber_motion

### LINKED ISSUE #2824: ImportError: Could not find library zaber-motion-lib-windows-amd64.dll in bindings
This issue similar to https://github.com/Nuitka/Nuitka/issues/2712 for the latest version of [zaber-motion](https://pypi.org/project/zaber-motion/). Until this is resolved, can you please suggest the content of the YAML I need to use? I tried this, https://github.com/Nuitka/Nuitka/issues/2712#issuecomment-1962080763, but it doesn't work. 

Python 3.11.9
Nuitka 2.1.6
zaber_motion 5.2.0

_Error when running .exe file:_

```powershell
PS C:\Users\abcd\build\zaber_import_issue.dist> .\zaber_import_issue.exe
Traceback (most recent call last):
  File "C:\Users\abcd\build\ZABER_~1.DIS\zaber_import_issue.py", line 1, in <module>
  File "C:\Users\abcd\build\ZABER_~1.DIS\zaber_motion\__init__.py", line 6, in <module zaber_motion>
  File "C:\Users\abcd\build\ZABER_~1.DIS\zaber_motion\library.py", line 4, in <module zaber_motion.library>
  File "C:\Users\abcd\build\ZABER_~1.DIS\zaber_motion\call.py", line 12, in <module zaber_motion.call>
  File "C:\Users\abcd\build\ZABER_~1.DIS\zaber_motion\bindings.py", line 39, in <module zaber_motion.bindings>
  File "C:\Users\abcd\build\ZABER_~1.DIS\zaber_motion\bindings.py", line 35, in _load_library
ImportError: Could not find library zaber-motion-lib-windows-amd64.dll in bindings. Please contact Zaber support.
PS C:\Users\abcd\build\zaber_import_issue.dist>
```

_Test code in zaber_import_issue.py_
```python
from zaber_motion.ascii import Connection

print("Loading...")
connection = Connection.open_serial_port(port_name='COM4')
print("Done...")
```

_Compile command:_ 
```powershell
python -m nuitka --standalone --output-dir=build .\zaber_import_issue.py
```

### PR DESCRIPTION
**Pull request by KRRT7**
fixes #2824

### CODE REVIEW

File: nuitka/plugins/standard/standard.nuitka-package.config.yml [hunk 1]
```diff
@@ -377,12 +377,12 @@
...
       - 'templates/*.j2'
 
-- module-name: 'bindings' # checksum: 7e84029d
+- module-name: 'bindings' # checksum: 95b3b99e
   dlls:
     - from_filenames:
         prefixes:
           - 'zaber-motion-lib'
-      when: 'None is not version("zaber_motion") >= (5,1,2)'
+      when: 'None is not version("zaber_motion") == (5,1,2)'
```
[the remark this answers is not in the document]
Comment (KRRT7) [reply nuitka/plugins/standard/standard.nuitka-package.config.yml:385]: yes, that is the intended change.

File: nuitka/plugins/standard/standard.nuitka-package.config.yml [hunk 2]
```diff
@@ -7392,14 +7392,24 @@
       when: 'macos and version("zaber_motion") < (5,1,2)'
     - depends:
         - 'bindings'
-      when: 'version("zaber_motion") >= (5,1,2)'
+      when: 'version("zaber_motion") == (5,1,2)'
```
[the remark this answers is not in the document]
Comment (KRRT7) [reply nuitka/plugins/standard/standard.nuitka-package.config.yml:7395]: intended change.

File: nuitka/plugins/standard/standard.nuitka-package.config.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kayhayen) [reply nuitka/plugins/standard/standard.nuitka-package.config.yml:385]: You mean like, it was the only version to be like that, and they reverted to previous behavior?

File: nuitka/plugins/standard/standard.nuitka-package.config.yml [hunk 3]
```diff
@@ -7392,14 +7392,24 @@
...
     - description: 'normalize path for DLLs'
       replacements_plain:
         'os.path.join(os.path.dirname(__file__), "..", "bindings", lib_name)': 'os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "bindings", lib_name))'
 
+- module-name: 'zaber_motion_bindings' # checksum: 679f33dc
+  dlls:
+    - from_filenames:
+        prefixes:
+          - 'zaber-motion-lib'
+      when: 'None is not version("zaber_motion") >= (5,1,3)'
```
Comment (kayhayen) [inline nuitka/plugins/standard/standard.nuitka-package.config.yml:7411]: Ok, I see that works, and `bindings` was really the top level name, I now remember, but apparently they gave up on it quickly, I need to revert my stupid commit on factory.

File: nuitka/plugins/standard/standard.nuitka-package.config.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (KRRT7) [reply nuitka/plugins/standard/standard.nuitka-package.config.yml:385]: > You mean like, it was the only version to be like that, and they reverted to previous behavior?

yes, i opened an issue in their repo and they mentioned that that specific version had unintended behavior, then they updated to the behavior they actually wanted.

**Comment by kayhayen** [comment]
I manually merged it.

**Pull request closed without merging**